UM-USElab/gradecraft-development
# 1533 grading resubmissions

Displays a timeline for students and professors that details the submission history and where a grade was received.

Here is a screenshot of what the timeline view looks like:

![screen shot 2016-01-20 at 1 16 09 pm](https://cloud.githubusercontent.com/assets/35017/12458549/5743cfd0-bf78-11e5-8a2b-ed55bc5e2657.png)

It is displayed in a "Submission History" tab for the assignment view for students and the submission show view for professors.

To test:

1. Log in as a student and create a submission for an assignment that allows for resubmissions
1. View the submission history and it should display that you created the submission
1. Change the submitted submission
1. View the submission history and it should display that you changed the submission
1. Log in as a professor and grade that submission
1. View the submission history and it should display that you graded the submission
1. Log in as a student and change a submission
1. View the submission history and it should display that you changed the submission
1. Log in as a professor and grade that submission
1. View the submission history and it should display that you graded the submission again

Closes #1533
